.kokoro/presubmit/downstream-build.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+
+function modify_shared_config() {
+  xmllint --shell pom.xml <<EOF
+  setns x=http://maven.apache.org/POM/4.0.0
+  cd .//x:artifactId[text()="google-cloud-shared-config"]
+  cd ../x:version
+  set ${SHARED_CONFIG_VERSION}
+  save pom.xml
+EOF
+}
+
+if [ -z "${MODULES_UNDER_TEST}" ]; then
+  echo "MODULES_UNDER_TEST must be set to run downstream-build.sh"
+  exit 1
+fi
+
+### Round 1
+## Get the directory of the build script and install java-shared-config
+scriptDir=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
+## cd to the parent directory, i.e. the root of the git repo
+cd "${scriptDir}/../.."
+mvn -B -ntp install -Dcheckstyle.skip -Dfmt.skip -DskipTests
+SHARED_CONFIG_VERSION=$(sed -e 's/xmlns=".*"//' pom.xml | xmllint --xpath '/project/version/text()' -)
+
+# Use GCP Maven Mirror
+mkdir -p "${HOME}/.m2"
+cp settings.xml "${HOME}/.m2"
+
+### Round 2
+git clone "https://github.com/googleapis/gapic-generator-java.git" --depth=1
```

Comment:
does this need to be udpated to googleapis/sdk-platform-java?

Reply:
Thanks Alice! The original iteration of the script was copied over from sdk-platform-java and was doing some extra work with using the SNAPSHOT version of shared-dependencies which may not be needed for the purpose of testing shared-config. I've cleaned this up a little.